### DISCUSSION TITLE
source-controller fails producing HelmChart

### DISCUSSION THREAD

**Discussion by nicogl** [fluxcd/flux2#955] [Q&A] [answered]
I'm defining a GitRepository chart source as per official [documentation](https://toolkit.fluxcd.io/guides/helmreleases/#git-repository), so I considered the caveat mentioned there: "There is one caveat you should be aware of: to make the source-controller produce a new chart artifact, the version in the Chart.yaml of the chart must be bumped."

However, looking at the "source-controller" pod logs I found errors like:

{"level":"error","ts":"2021-02-19T21:08:31.664Z","logger":"controller.helmchart","msg":"Reconciler error","reconciler group":"source.toolkit.fluxcd.io","reconciler kind":"HelmChart","name":"flux-system-am-helmrelease","namespace":"flux-system","error":"load chart error: validation: chart.metadata.version is required"}

I tried three different options in my Chart.yaml file:

1) `version: ""`
2) `version: `
3) `#version: commented line`

This error is preventing the artifact from being generated, and therefore the source-controller doesn't apply/refresh any change I make on my application.
Am I missing something here? Any help will be highly appreciated!

Regards, Nicolas

**Comment by stefanprodan** [comment, marked as answer]
An empty version means no version, you need to provide a valid semver as the value.
